folding-schemes/src/folding/nova/serialize.rs

```diff
@@ -204,21 +206,28 @@ pub mod tests {
         let (cf_pedersen_params, _) = Pedersen::<Projective2>::setup(&mut rng, cf_cs_len).unwrap();
 
         // Initialize nova and make multiple `prove_step()`
-        type NOVA<CS1, CS2> =
-            Nova<Projective, GVar, Projective2, GVar2, CubicFCircuit<Fr>, CS1, CS2>;
+        type NOVA = Nova<
```

Comment:
I'd avoid CAPS types. As CAPS are reserved for either constants or thinks that we pass as parameters that we want to declare similar to LateX.

Reply:
```suggestion
        type NovaInstance = Nova<
```

WDYT? Not sure either.

Reply:
this is not an instance (in the sense that we use 'instance' in the rest of the repo), but agree on the caps issue, will update all the types that use caps to avoid them